kpatch-build/create-diff-object.c

```diff
@@ -2100,10 +2101,17 @@ static int printk_index_group_size(struct kpatch_elf *kelf, int offset)
 
 static int parainstructions_group_size(struct kpatch_elf *kelf, int offset)
 {
+	struct utsname name;
 	static int size = 0;
 	char *str;
+	int major, minor;
 
-	if (!size) {
+	if (uname(&name) < 0) {
```

Comment:
Unfortunately create-diff-object can't assume that it's building for the currently running kernel.  (Think about how it operates when kpatch-build is passed the -r, -s, -v options.)